src/main/java/core/basesyntax/file/FileReaderImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.file;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+
+    @Override
+    public List<String> read(String fileName) {
+        List<String> data = new ArrayList<>();
+        try (BufferedReader reader = new BufferedReader(new java.io.FileReader(fileName))) {
+            String line;
+            reader.readLine();
```

Comment:
The line `reader.readLine();` skips the first line of the file. If this is not the intended behavior, you should remove this line. If the first line is a header and should be skipped, then this is correct.